lib/generapp/generators/app_generator.rb

```diff
@@ -2,18 +2,30 @@
 require 'rails/generators/rails/app/app_generator'
 
 module Generapp
-  module Generators
+  module Generators #:nodoc
+    # Rails App generator
     class AppGenerator < ::Rails::Generators::AppGenerator
-      class_option :database, type: :string, aliases: '-d', default: 'postgresql',
-                   desc: "Configure for selected database (options: #{DATABASES.join("/")})"
-
-      class_option :skip_test_unit, type: :boolean, aliases: '-T', default: true,
+      class_option :database,
+                   type: :string,
+                   aliases: '-d',
+                   default: 'postgresql',
+                   desc: "Configure for selected database (options: #{DATABASES.join('/')})"
+
+      class_option :skip_test_unit,
+                   type: :boolean,
+                   aliases: '-T',
+                   default: true,
                    desc: 'Skip Test::Unit files'
 
-      class_option :skip_turbolinks, type: :boolean, default: true,
+      class_option :skip_turbolinks,
+                   type: :boolean,
+                   default: true,
                    desc: 'Skip turbolinks gem'
 
-      class_option :skip_bundle, type: :boolean, aliases: '-B', default: true,
+      class_option :skip_bundle,
+                   type: :boolean,
+                   aliases: '-B',
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.